js/map.js

```diff
@@ -0,0 +1,123 @@
+//import L from 'leaflet';
+/* global L:readonly */
+import {getCardsArray} from './data.js';
+import {getPopUp} from './popup.js';
+
+
+const CITY_COORDINATE_X = 35.40480;
+const CITY_COORDINATE_Y = 139.45360;
+const FINAL_ARRAY_ELEMENTS = 10;
+const MAIN_ICON_SIZE_WIDTH = 52;
+const MAIN_ICON_SIZE_HEIGHT = 52;
+const MAIN_ICON_ANCHOR_SIZE_WIDTH = 26;
+const MAIN_ICON_ANCHOR_SIZE_HEIGHT = 52;
+const MAIN_MARKER_DEFAULT_COORDINATE_X = 35.40480;
+const MAIN_MARKER_DEFAUL_COORDINATE_Y = 139.45360;
```

Comment:
Заодно опечатку поправь
```suggestion
const MAIN_MARKER_DEFAULT_COORDINATE_Y = 139.45360;
```